browser/html/framed.doc.html

```diff
@@ -476,6 +482,7 @@ <h3>Various other messages to post</h3>
             <button onclick="startPresentation(); return false;">Start presentation</button>
 
             <button onclick="exportAsHtml(); return false;">Export as HTML</button>
+            <button onclick="exportAsPdfWithNotify(); return false;">Export as PDF with Notify set</button>
             <button onclick="getExportFormats(); return false;">Get export formats</button>
```

Comment:
I would add an empty `<div>` here to make the next button pushed to the next row. This would keep the logical grouping.

Reply:
done